### ISSUE TITLE
Filter trips by month

### ISSUE THREAD

**Issue by jahlela** [Ledgy/TrainTrack#7]
**Description**
* Currently, all trips are shown in the trip list. 
* Add a button that allows the user to filter by "Month", instead of just "Week"

**Specs**
* This shows the eventual result for filtering by week, month, and year. However, this task is **only adding filtering by MONTH**
* Mockup: 

<img width="234" alt="Screen Shot 2019-10-21 at 11 48 20 AM" src="https://user-images.githubusercontent.com/6724153/67195153-bffd5380-f3f8-11e9-86db-d13d35f825f9.png">

<img width="524" alt="Screen Shot 2019-10-21 at 11 53 35 AM" src="https://user-images.githubusercontent.com/6724153/67195570-73664800-f3f9-11e9-869a-c66f21307bc5.png">

**Comment by jaredgreen** [comment]
I will give filter by month a try

**Comment by jahlela** [comment]
@jaredgreen Definitely. You may want to coordinate with @n1tin on this similar task: https://github.com/Ledgy/TrainTrack/issues/6
